README.md

```diff
@@ -216,6 +216,16 @@ yarn; yarn build; DEBUG=electron-builder yarn release --win;
 ./packages/fether-electron/dist/Parity\ Fether-0.3.0.exe
 ```
 
+# Development
```

Comment:
Could we move this to the wiki instead?

Reply:
yes good idea, i've removed it and will add to wiki if necessary